src/Npgsql/NpgsqlConnection.cs

```diff
@@ -993,7 +993,25 @@ public int ProcessID
         /// <remarks>
         /// See https://www.postgresql.org/docs/current/static/sql-copy.html.
         /// </remarks>
-        public NpgsqlBinaryImporter BeginBinaryImport(string copyFromCommand)
+        public NpgsqlBinaryImporter BeginBinaryImport(string copyFromCommand) =>
```

Comment:
nit: we usually put the `=>` on the beginning of the next line (more immediately visible)